package.json

```diff
@@ -35,6 +35,9 @@
   },
   "author": "Juan Cruz Viotti <juan@resin.io>",
   "license": "Apache-2.0",
+  "engines": {
+    "node": ">=6.0"
```

Comment:
So it looks like that this enforces node v6+ which is a bit different than `Stop actively supporting node 4`.

Reply:
This isn't enforced by anybody (as far as I know), it's [advisory only](https://docs.npmjs.com/files/package.json#engines). It makes npm print a warning if you install it using an unsupported the wrong node version, but no more.